### PR TITLE
[editorial] Describe barycentric coordinates

### PR DESCRIPTION
**Pull request by kvark**
Continuing from #2162

***
<a href="https://pr-preview.s3.amazonaws.com/kvark/gpuweb/pull/2370.html" title="Last updated on Dec 3, 2021, 6:06 PM UTC (00e719f)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/gpuweb/gpuweb/2370/8ea3aea...kvark:00e719f.html" title="Last updated on Dec 3, 2021, 6:06 PM UTC (00e719f)">Diff</a>

### CODE REVIEW

File: spec/index.bs [hunk 1]
```diff
@@ -9458,61 +9456,114 @@ The coverage mask depends on multi-sampling mode:
...
 
-The next step is determining a set of [=fragments=] inside the polygon in framebuffer space -
-these are locations scheduled for the per-fragment operations.
-The determination is based on |descriptor|.{{GPURenderPipelineDescriptor/multisample}}:
-<dl class="switch">
-    : disabled
-    :: [=Fragment=]s are associated with pixel centers. That is, all the points with coordinates |C|, where
-        fract(|C|) = vector2(0.5, 0.5) in the [=framebuffer=] space, enclosed into the polygon, are included.
-        If a pixel center is on the edge of the polygon, whether or not it's included is not defined.
+    1. Determine if the polygon facing,
```
Comment (kainino0x) [inline spec/index.bs:9506]: "determine if the polygon is front-facing"?

**Review by kainino0x — APPROVED** [review]
Sorry for the delay, lgtm

**Review by toji — APPROVED** [review]
LGTM as well (with the typo Kai pointed out fixed)

**Comment by kvark** [comment]
Thanks for the reviews!

One tricky bit about how barycentrics are defined is the polygon shape. The provided equations are only for triangles, but at the stage where we need barycentrics, we have polygons with more vertices potentially, resulting from view clipping.
So in this PR I'm trying to describe them as generic as possible over the polygon shape, and I'm saying that the exact implementation for non-triangles is implementation-dependent. This matches Vulkan overall, but I think it makes less assumptions about having a triangle.

**Pull request merged**